packages/io/__tests__/listener.ts

```diff
@@ -51,6 +52,12 @@ describe('test Listener', () => {
       }),
     getCurrentEpoch: () => Promise.resolve(mockEpoch),
     getBlock: () => Promise.resolve(mockBlock),
+    getAllLiveCellsWithWitness: function (
```

Comment:
Set `_lockScript` prior to `_typeScript` if `_typeScript` is optional, then the method could be called as `getAllLiveCellsWithWitness(lock)` instead of `getAllLiveCellsWithWitness(, lock)` if `type script` is `undefined`.

And `_typeScript: Script | undefined` could be declared as `_typeScript?: Script`